src/components/MapSelectors/datasets/us_states_dataset_01_02_2020.json

```diff
@@ -3,318 +3,371 @@
     {
       "state_code": "AK",
       "state": "Alaska",
+      "state_url_name": "alaska-ak",
```

Comment:
is it weird that we use dashes in the state_url_name but underscores in county_url_name ?

I don't care much though if this is intentional.